### DISCUSSION TITLE
如何限定请求地址？

### DISCUSSION THREAD

**Discussion by wukaipeng-dev** [labring/laf#124] [Feedbacks] [answered]
目前有一个需求，是从前端请求某个laf云函数，需要在该云函数中限定请求的地址，比如只有来自example.com或者example.cn的请求才会被继续处理，请问应该如何实现？

**Comment by maslow** [comment, marked as answer]
> 目前有一个需求，是从前端请求某个laf云函数，需要在该云函数中限定请求的地址，比如只有来自example.com或者example.cn的请求才会被继续处理，请问应该如何实现？

在云函数中可以获取 http header，可以获取到请求源（若存在）： "referer": "https://www.lafyun.com/"

console.log(ctx.headers)
